### PR TITLE
Url param

### PR DESCRIPTION
**Pull request by LoganWright**
#1055

### CODE REVIEW

File: Sources/Vapor/FormURLEncoded/StructuredData+FormURLEncoded.swift [hunk 1]
```diff
@@ -113,15 +113,40 @@ extension Node {
...
         for (key, val) in dict {
             var subbytes: [Byte] = []
-            
-            subbytes += key.urlQueryPercentEncoded.makeBytes()
-            subbytes += Byte.equals
-            subbytes += val.string?
-                .urlQueryPercentEncoded.makeBytes() ?? []
-            
+
+            if let object = val.object {
```
Comment (tanner0101) [inline Sources/Vapor/FormURLEncoded/StructuredData+FormURLEncoded.swift]: we should also support arrays while we are here. 

i believe you can do arrays just like you can in html but i'm not 100% sure

```
foo[]=1&foo[]=2 // foo = [1,2]
```

Comment (LoganWright) [inline Sources/Vapor/FormURLEncoded/StructuredData+FormURLEncoded.swift]: Oh yea, good call, I'm pretty sure parser works there, I'll add it into serializer.

Comment (LoganWright) [inline Sources/Vapor/FormURLEncoded/StructuredData+FormURLEncoded.swift]: @tanner0101 ready for second look 👍

Comment (tanner0101) [inline Sources/Vapor/FormURLEncoded/StructuredData+FormURLEncoded.swift]: looks like the changes didn't get pushed

Comment (LoganWright) [inline Sources/Vapor/FormURLEncoded/StructuredData+FormURLEncoded.swift]: @tanner0101 push got merge blocked, forgot to resolve. up now

**Pull request merged**